doc/metadata_routing.rst

```diff
@@ -265,6 +265,10 @@ Meta-estimators and functions supporting metadata routing:
 - :class:`sklearn.multioutput.MultiOutputRegressor`
 - :class:`sklearn.multioutput.RegressorChain`
 - :class:`sklearn.pipeline.Pipeline`
+- :class:`sklearn.linear_model.ElasticNetCV`
+- :class:`sklearn.linear_model.LassoCV`
+- :class:`sklearn.linear_model.MultiTaskElasticNetCV`
+- :class:`sklearn.linear_model.MultiTaskLassoCV`
```

Comment:
I'd move these up, this list is alphabetically sorted by module/classs name, so I'd keep it sorted.